### DISCUSSION TITLE
Script for colorizing folder icons to match Gradience preset

### DISCUSSION THREAD

**Discussion by modcolocko** [GradienceTeam/Gradience#804] [Show and tell]
I mainly created this script for my own needs, but I hope it comes in handy for a few of you.

It's quite bodged frankly, the icons come out transparent and muted, but they still match better than the default Adwaita icons for most themes. It also doesn't use the current icons, because of the fact that the newest version of the Adwaita icon theme use scalable icons for all the folders. And I built this script before that change.

(i might eventually update it so it can use the newer icons)

https://gitlab.com/modcolocko/adwaita-icon-colorizer

**Comment by 0xMRTT** [comment]
Thanks!!!!
